worker/src/test/java/gov/cms/ab2d/worker/processor/BundleUtils.java

```diff
@@ -33,9 +45,27 @@ public static Bundle createBundle(Bundle.BundleEntryComponent ... bundleEntries)
     }
 
     public static Patient createPatient(String patientId, String mbi, int year) {
+        return createPatient(patientId, mbi, year, true);
+    }
+
+    public static Patient createPatientWithMultipleMbis(String patientId, int numMbis, int year) {
+        var patient = new Patient();
+        patient.getIdentifier().add(createBeneficiaryIdentifier(patientId));
+
+        patient.getIdentifier().add(createMbiIdentifier(patientId + "mbi-" + 0, true));
+
+        for (int i = 1; i < numMbis; i++) {
```

Comment:
nitpick
          i => idx

It's a whole lot easier to search for 3 letter loop indexes rather than 1.